wagtail/contrib/modeladmin/views.py

```diff
@@ -283,7 +283,7 @@ def to_row_dict(self, item):
             f, attr, value = lookup_field(field, item, self.model_admin)
             if not value:
                 value = getattr(attr, 'empty_value_display', self.model_admin.get_empty_value_display(field))
-            row_dict[field] = value
+            row_dict[field] = force_str(value)
```

Comment:
If you see https://github.com/wagtail/wagtail/blob/master/wagtail/admin/views/reports.py, `force_str` is currently chosen as the default in `get_preprocess_function` anyway, which allows the normal `SpreadsheetExportMixin` overrides (by value type and by field name) to be used, whereas adding it here would prevent those from taking place. I'd suggest removing it here.